rfps/open/formal_guarantees_for_grandpa.md

```diff
@@ -27,9 +26,9 @@ The deliverables listed below are just an initial draft, assuming the project us
 | 0a. | License | Apache 2.0 / MIT / Unlicense |
 | 0b. | Documentation | A document describing the design decisions in the modeling process, including justification for abstractions and assumptions (e.g. on the network model, latency, behavior of validators, nature of faults) w.r.t to protocol in the Paper and Spec. In-line comments in the TLA+/ PlusCal models that provide a clear mapping to the feature/property. |
  0c. | Testing Guide | Instructions to set up the required environment to run the analysis, preferably a docker image with all the tools pre-installed. | 
-| 0d. | Article | High-level document summarizing the results of the verification efforts as well as a final presentation for a broader audience that summarizes the key take-aways from the verification. |
-| 1 | Proof | Models and specs in TLA+ that adhere to protocol implementations with reasonable abstraction gaps. As a stepping stone, the block production mechanism could be instantiated with BABE. | 
-| 2 | Proof | Formalize the validity, finality and liveness of GRANDPA as temporal properties in TLA+.  | 
-| 3 | Engagements | Engage with the Web3 research team via regular meetings to refine the models and specs. For eg., clarify any assume/ rely reasoning made in the protocols.| 
-
+| 0d. | Article | High-level document summarizing the results of the verification efforts as well as a final presentation for a broader audience that summarizes the key take-aways. |
+| 1 | Proof Artifact| Models and specs in TLA+ that adhere to protocol implementations with reasonable abstraction gaps. As a stepping stone, the block production mechanism could be instantiated with BABE. | 
+| 2 | Proof Artifact| Formalize the validity, finality and liveness of GRANDPA as temporal properties in TLA+.  | 
+| 3 | Proof Artifact | Proof of correctness can be delivered in stages. Verification of safety properties is mandatory and liveness properties is an optional objective. |
```

Comment:
I removed this because it's not a clear requirement. Maybe you can add this after "The deliverables listed below are just an initial draft, assuming the project uses the approach of Modelcheking in TLA+ and can be changed."

Reply:
Sure, make sense, updated accordingly